src/components/AntiPatternTagsComponent.vue

```diff
@@ -10,6 +10,11 @@
             This website serves as a knowledge base for Service-Based Antipatterns
             and has been created due to a study project at the University of Stuttgart.
         </v-card-text>
+        <v-subheader>Sorting</v-subheader>
+        <v-radio-group v-model="value.sorting">
+            <v-radio label="Name" value="name"/>
```

Comment:
I don't like a radio group since we can't do backward sorting. 
Option A:
We could use a single dropdown with all the filter options spelled out e.g.
`sorting ["name: A-Z", "name: Z-A", "evidence: low-high", "evidence: high-low"]`
Option B:
We could use multiple dropdowns with only the forward and backward sorting option of one type
`sorting by: name ["name: A-Z", "name: Z-A"], evidence ["evidence: low-high", "evidence: high-low"]`

Option A should be sufficient for most use cases while option B provides additional support to use stacked filters.
@tschechlovdev what do you think?